nipype/algorithms/tests/test_auto_CompCor.py

```diff
@@ -1,37 +0,0 @@
-# AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
-from __future__ import unicode_literals
-from ..confounds import CompCor
-
-
-def test_CompCor_inputs():
```

Comment:
@satra Thanks! 

$ pip install --upgrade https://github.com/satra/nipype/archive/fix/bru2.zip
$ python -c "import nipype; nipype.test()"
...
===== 585 passed, 1 skipped in 22.58 seconds ====